scale_build/utils/logger.py

```diff
@@ -0,0 +1,13 @@
+import logging
+import os
+
+from .paths import LOG_DIR
+
+
+def get_logger(logger_name, logger_path=None, mode='a+'):
```

Comment:
Where is `logger_path=None` used? What will happen in `os.path.join(LOG_DIR, logger_path)` if `logger_path` is `None`?